### DISCUSSION TITLE
Error connecting to codespace with VS Code with the extension

### DISCUSSION THREAD

**Discussion by nikneem** [community/community#3647] [Codespaces]
When I have my codespace open and click "Open in VS Code" from the menu it takes ages to load my project in VS Code and in the en fails. I've tried this a couple of times.

![image](https://user-images.githubusercontent.com/11193956/112886208-e6391f80-90d1-11eb-87f5-cbf9b2db1105.png)

**Comment by PranamBhat** [comment]
Hi @nikneem 

Looks like my remote server disabled port forwarding for certain accounts. 


Check this out : 

[https://github.com/microsoft/vscode-remote-release/issues/103](https://github.com/microsoft/vscode-remote-release/issues/103)


[https://github.com/microsoft/vscode-remote-release/issues/103#issuecomment-497118857](https://github.com/microsoft/vscode-remote-release/issues/103#issuecomment-497118857)


[https://code.visualstudio.com/docs/remote/troubleshooting#_enabling-alternate-ssh-authentication-methods](https://code.visualstudio.com/docs/remote/troubleshooting#_enabling-alternate-ssh-authentication-methods)
